cap/modules/services/views/zenodo.py

```diff
@@ -38,39 +38,41 @@ def _get_zenodo_record(zenodo_id):
     url = "{}/records/{}".format(zenodo_server_url, zenodo_id)
     params = {"access_token": current_app.config.get('ZENODO_ACCESS_TOKEN')}
 
-    resp = requests.get(url, headers={'Content-Type': 'application/json'},
+    resp = requests.get(url,
+                        headers={'Content-Type': 'application/json'},
                         params=params)
-    return resp.json(), resp.status_code
+
+    return jsonify(resp.json()), resp.status_code
```

Comment:
why changed? better have `jsonify` on the view method, so this method can be reusable